trainer:
  batch_dim: 1024
  memory_len: 1000000
  period_roll: 256
  force_threshold: 5.0
  w_orientation: 0.3
  w_angular: 0.3
  w_linear: 0.2
  w_force_balance: 0.1
  w_joint_velocity: 0.1
  w_height: 0.3